build.sbt

```diff
@@ -49,12 +51,11 @@ lazy val `scalatestplus-play` = project
       "org.scalatest" %% "scalatest" % ScalatestVersion,
       "org.seleniumhq.selenium" % "selenium-java" % SeleniumVersion exclude(org = "com.codeborne", name = "phantomjsdriver"),
       "org.seleniumhq.selenium" % "htmlunit-driver" % HtmlUnitVersion,
+      "net.sourceforge.htmlunit" % "htmlunit-cssparser" % CssParserVersion,
       "com.codeborne" % "phantomjsdriver" % PhantomJsDriverVersion,
       "com.typesafe.play" %% "play-test" % PlayVersion,
       "com.typesafe.play" %% "play-ws" % PlayVersion,
-      "com.typesafe.play" %% "play-ahc-ws" % PlayVersion,
-      "com.typesafe.play" %% "play-cache" % PlayVersion % Test,
```

Comment:
This is an ancient dependency. Play cache used to be one of the core dependencies in Play. We split it to a new dependency in 2.6 and had to re-add it here. But it was not necessary. The only reference made to `EhCacheModule` was to disable it.

So we can now remove it.